containers/dgxie/docker-compose.yml

```diff
@@ -1,10 +1,11 @@
 # sudo docker run --rm -ti --net=host --privileged -v /mnt/:/iso:ro -e HOST_INT_PUB=ens160 -e HOST_INT_PRV=ens192 --name dgxie dgxie
 # sudo docker run -d --name nfs --privileged --net=host -v /home/ubuntu/:/master -e SHARED_DIRECTORY=/master nfs-server:latest
-version: '3'
+version: '3.4'
 services:
     dgxie:
         build:
             context: .
+            network: "host"
```

Comment:
This was required when running on the base GCP Ubuntu image and a few other boxes.